cave/com.raytheon.uf.viz.d2d.ui/plugin.xml

```diff
@@ -1452,6 +1452,24 @@
             schemeId="com.raytheon.viz.ui.cave.scheme"
             sequence="M1+H">
       </key>
+      <key
+            contextId="com.raytheon.uf.viz.d2d.ui"
+            commandId="com.raytheon.viz.ui.loadSerialized"
+            schemeId="com.raytheon.viz.ui.cave.scheme"
+            sequence="M1+1">
+      </key>
+       <key
+            contextId="com.raytheon.uf.viz.d2d.ui"
+            commandId="com.raytheon.viz.ui.saveProcedure"
+            schemeId="com.raytheon.viz.ui.cave.scheme"
+            sequence="M1+2">
```

Comment:
Change to M1+S (Ctrl+S)

Tested before making the change and this key binding didn't do anything, after changing it opened the save bundle GUI